docs/element plus.md

```
@@ -0,0 +1,46 @@
+
+在项目根目录打开shell，执行下面命令生成物料文件：
+```shell
+node ./scripts/genMaterial.js
+```
+![](./images/01.png)
+
+物料文件bundle.json会自动生成到根目录中
+
+### 物料文件使用
+将生成的物料文件bundle.json重命名为elementplus.json,我们再packages\design-core\config目录下lowcode.config.js的文件中material中的'/mock/bundle.json'修改为'/mock/elementplus.json'。
+![](./images/04.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)